pkg/controller/volume/persistentvolume/scheduler_bind_cache_metrics.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package persistentvolume
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"time"
+)
+
+// VolumeSchedulerSubsystem - subsystem name used by scheduler
+const VolumeSchedulerSubsystem = "volumeScheduler"
+
+var (
+	VolumeBindingAddToSchedulerBinderCache = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Subsystem: VolumeSchedulerSubsystem,
+			Name:      "total_number_add_to_scheduler_binder_cache",
```

Comment:
Please see Prometheus naming best practices: https://prometheus.io/docs/practices/naming/
Names should have a single word prefix. You may want to start all these names with `volume`. Also, move `total` to the end.
Please fix other metric names similarly.